release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -550,6 +550,21 @@ In this release you can now specify the transmit hash policy for load balancing
 
 For more information, see xref:../networking/multiple_networks/secondary_networks/creating-secondary-nwt-other-cni.adoc#nw-multus-bond-cni-object_configuring-additional-network-cni[Configuration for a Bond CNI secondary network].
 
+[id="ocp-4-20-networking-namespaced--sriov-app-owners_{context}"]
+==== Directly manage your SR-IOV Networks in your application namespace
+
+This release introduces namespaced SR-IOV networks, allowing application owners to directly create and manage their own SR-IOV networks within their application namespaces. This feature gives you greater control over your network configurations and helps simplify your workflow. 
```

Comment:
There's new guidance about release note format here: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-features-enhancements

I've added a suggest re-write below to align a bit more with the new format. And also removes anthropomorphism: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism